### PR TITLE
Testing branch

### PR DESCRIPTION
**Pull request by huiminwu**

### CODE REVIEW

File: src/main/java/frc/robot/commands/LiftMoveCommand.java [hunk 1]
```diff
@@ -32,14 +31,22 @@ protected void initialize() {
...
-        if (targetLevel == Level.LEVEL_ZERO) {
-            Robot.lift.move(Robot.oi.operatorGamepad.getLeftY());
+    int level = 0;
+        if(level == 0) {
+            double speed = Robot.oi.operatorGamepad.getLeftY();
+            if(Math.abs(speed) > .2) {
+                Robot.lift.moveLift(speed);
+                System.out.println("Speedy" + speed);
+            } else {
+                Robot.lift.moveLift(0);
```
Comment (justinkim) [inline src/main/java/frc/robot/commands/LiftMoveCommand.java]: There's a `stopLift()` method.

File: src/main/java/frc/robot/subsystems/Lift.java [hunk 2]
```diff
@@ -116,31 +115,33 @@ private double rampMultiplier(double distance) {
...
+    //         // If you want to move the lift up, get the distance from the top and adjust speed proportionally.
+    //         double distanceFromTop = RobotMap.LIFT_MAX_HEIGHT - currentHeight;
+    //         speed = rampMultiplier(distanceFromTop) * desiredSpeed;
+    //         speed = Math.max(speed, RobotMap.LIFT_MIN_SPEED);
+    //     }
+    //     // If the current height isn't within the height range for ramping, move without ramping.
+    //     forceMoveNoRamp(speed);
+    // }
+
+    public void moveLift(double speed) {
```
Comment (justinkim) [inline src/main/java/frc/robot/subsystems/Lift.java]: `moveLift`?

File: src/main/java/frc/robot/commands/LiftMoveCommand.java (hunk 1 above)
Comment (huiminwu) [reply src/main/java/frc/robot/commands/LiftMoveCommand.java]: refractored stopLift to stop and used that method here

File: src/main/java/frc/robot/subsystems/Lift.java (hunk 2 above)
Comment (huiminwu) [reply src/main/java/frc/robot/subsystems/Lift.java]: refractored to move

**Pull request merged**